src/Admin/AdminConsole/Models/ProviderViewModel.cs

```diff
@@ -1,24 +1,82 @@
 ﻿using Bit.Core.AdminConsole.Entities.Provider;
 using Bit.Core.AdminConsole.Enums.Provider;
 using Bit.Core.AdminConsole.Models.Data.Provider;
+using Bit.Core.Billing.Entities;
+using Bit.Core.Billing.Enums;
 
 namespace Bit.Admin.AdminConsole.Models;
 
 public class ProviderViewModel
 {
     public ProviderViewModel() { }
 
-    public ProviderViewModel(Provider provider, IEnumerable<ProviderUserUserDetails> providerUsers, IEnumerable<ProviderOrganizationOrganizationDetails> organizations)
+    public ProviderViewModel(
+        Provider provider,
+        IEnumerable<ProviderUserUserDetails> providerUsers,
+        IEnumerable<ProviderOrganizationOrganizationDetails> organizations,
+        IReadOnlyCollection<ProviderPlan> providerPlans)
     {
         Provider = provider;
         UserCount = providerUsers.Count();
         ProviderAdmins = providerUsers.Where(u => u.Type == ProviderUserType.ProviderAdmin);
-
         ProviderOrganizations = organizations.Where(o => o.ProviderId == provider.Id);
+
+        var usedEnterpriseSeats = ProviderOrganizations.Where(po => po.Plan.Contains("Enterprise")).Sum(po => po.OccupiedSeats);
```

Comment:
I suggest you handle the nullable value types here rather than allowing it to propagate down. e.g.

```suggestion
        var usedEnterpriseSeats = ProviderOrganizations.Where(po => po.Plan.Contains("Enterprise")).Sum(po => po.OccupiedSeats.GetValueOrDefault());
```

Reply:
Updated here: https://github.com/bitwarden/server/pull/5326/commits/abb8e88691adeb25dfe6ff805081f9e5790f2c47